cluster.yaml.example

```diff
@@ -143,3 +143,4 @@ openVPNServerResources:
           memory: 200Mi
 apiServerAuditEnabled: true
 restartDate: "2017-07-27T16:39:00Z"
+extraFeatureGates: [ExpandInUsePersistentVolumes=true,Accelerators=true,ExperimentalCriticalPodAnnotation=true]
```

Comment:
What is Accelerators?

I see no corresponding feature:
https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go

Keep in mind that OpenShift tightly controls what gates are enabled or disabled and we restrict upgradeability around the use of particular FeatureSets.

See:
https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L99

I would expect ROKS to use the same FeatureSet as  our default features list (both enabled and disabled):
https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113

Reply:
@derekwaynecarr we use `ExpandInUsePersistentVolumes=true` but the others I added just for an example. I can remove the others.